src/tinypeel/tinypeel.py

```diff
@@ -602,11 +625,12 @@ def main():
     )
     if not args.no_param:
         PeelingIO.writeOutParamaters(peelingInfo)
+    if not args.altalleleprobfile: # Only write alternative allele probabilities if user has **not** inputted them
+        PeelingIO.writeOutAltAlleleProb(pedigree, peelingInfo)
```

Comment:
I'm a bit confused, this output file should be output whenever user use the option `alt_allele_prob`. We need the output file when the estimation of the alternative allele frequency is used.

Reply:
This code will output the file when the default or estimated alternative allele probability is used. It will not produce an output file if the user has defined the alternative allele probability in the input. I thought it would save an additional command for the user. But, happy to add the option instead!

Reply:
AlphaPeel is working in a way that the users control all the additional output files, so I think it is better to keep it.